tests/python-gpu/test_gpu_gdf.py

```diff
@@ -0,0 +1,44 @@
+import numpy as np
+import pandas as pd
+try:
+    import cudf.dataframe as gdf
```

Comment:
cuDF references in Python need to be removed, and reworked to accept two cases:
1. a list of `__cuda_array_interface__` objects ... equivalent to a set of columns
2. a single `__cuda_array_interface__` object ... equivalent to accepting a matrix of values